docs/start/build-and-run/aot.md

```diff
@@ -121,7 +121,7 @@ This AOT-compiled WASM file is compatible with any WebAssembly runtime. However,
 
 <!-- prettier-ignore -->
 :::note
-On MacOS platforms, the universal WASM format will `bus error` in execution. By default, the `wasmedge compile` tool optimizes the WASM in the `O2` level. We are trying to fix this issue. For working around, please use the shared library output format instead.
+On MacOS platforms, the universal WASM format will `bus error` in execution. By default, the `wasmedge compile` tool optimizes the WASM in the `O2` level. We are trying to fix this issue. For working around, use the shared library output format instead.
```

Comment:
```suggestion
On MacOS platforms, the universal WASM format will `bus error` in execution. By default, the `wasmedge compile` tool optimizes the WASM in the `O2` level. We are trying to fix this issue. For working around, please use the shared library output format instead.
```

Reply:
Hi @adithyaakrishna, for this suggested change of adding back the word 'please,' I honestly think it's not a necessary change. The reason I made the change to remove the word 'please' from the sentence is because in technical writing, the word 'please' is considered redundant. Docs serve as strict instructions for users, so the place of 'please' in docs is questionable. I'll kindly request you do some research on using the word 'please' in technical writing and Docs generally, and we could have a final conclusion on this. If you also insist on adding 'please,' I'd love to know specifically your decision for using the word 'please' in the sentence. Thanks